modules/cnf-topology-aware-lifecycle-operator-installation-cli.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-talo-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="installing-topology-aware-lifecycle-operator-using-cli_{context}"]
+= Installing the Topology Aware Lifecycle Operator by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the Topology Aware Lifecycle Operator (TALO).
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the latest version of {rh-rhacm} Operator.
+* Set up a hub cluster with disconnected registry.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Namespace` custom resource (CR) for the TALO:
+.. Define the `Namespace` CR and save the YAML file, for example, `talo-namespace.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators
+----
+.. Create the `Namespace` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f talo-namespace.yaml
+----
+
+. Create an `OperatorGroup` CR:
```

Comment:
Please remove this part. OperatorGroup "global-operators" should have already existed. 

This is similar to how gitops operator is deployed, which is also under global-operators operatorgroup under openshift-operators namespace.